internal/support/logging/mongo.go

```diff
@@ -22,13 +24,18 @@ type mongoLog struct {
 	session *mgo.Session // Mongo database session
 }
 
-func connectToMongo() (*mgo.Session, error) {
+func connectToMongo(dic *di.Container) (*mgo.Session, error) {
```

Comment:
Don't pass the dic around. It's a god object intended to bridge the gap after bootstrap/init between routing and controller functions/methods. Pull out required dependencies in the caller and pass those dependencies explicitly to connectToMongo.